backend/backend/settings.py

```diff
@@ -47,6 +47,7 @@ def get_secret(secret_arn):
     "rest_framework_simplejwt.token_blacklist",
     "axes",
     "zygoat_django",
+    "zxcvbn"
```

Comment:
so this shouldn't be added here, we just want to install it. you'll want to add it to [this file](https://github.com/MetLifeLegalPlans/portunus/blob/develop/backend/requirements.txt)

this part of the file is telling django what other django apps we want to use. zxcvbn isn't a django app. let me know if that doesn't make sense!

Reply:
Okay make sense, to the requirement file we add the zxcvbn version right?

Reply:
yep!

i always pip install and use that to find a version number. `pip3.8 install zxcvbn` and then at the bottom of the install output it'll print out what version number you just installed. looks like it should be 4.4.28.